src/styles/main.scss

```diff
@@ -0,0 +1,138 @@
+.main {
+  padding: 50px;
+  display: grid;
+  grid-template-columns: repeat(auto-fill, $card-size);
+  gap: 46px 48px;
+  justify-content: center;
+}
+
+@media (min-width: $small-screen) {
+  .main {
+    grid-template-columns: repeat(2, $card-size);
+  }
+}
+
+@media (min-width: $media-screen) {
+  .main {
+    grid-template-columns: repeat(3, $card-size);
+  }
+}
+
+@media (min-width: $big-screen) {
+  .main {
+    grid-template-columns: repeat(4, $card-size);
+  }
+}
```

Comment:
All `@media` usage could (and probably should) be inside the `.main` selector [Reference](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules#:~:text=Write%20pseudo%2Dclass%2C%20pseudo%2Delement%20selectors%20inside%20general%20selector.%20As%20well%20as%20media%20queries.)